data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
```

Comment:
- [ ] This PR should be invalidated by the previous reviewer cause it doesn't have a professional readme and also it doesn't implement the Git flow correctly please make sure:warning:For Students :warning:
As a Code reviewer, Almost all the Pull requests that I receive are not Professional according to the General Requirements
Linters
Github Flow
should be documented in Professional way
Please make sure to use the new template of the Readme file and it should be customized to your own with all the sections except the QAF which is optional 
:bangbang:Please before submitting your project for review please make sure that your Pull Request is professional according to the above requirements
That will save both of us a lot of time and thank you for keeping reading until here